src/debug_menu.cpp

```diff
@@ -1017,7 +1018,7 @@ static void change_spells( Character &character )
             mvwprintz( w_name.window, point( 2, line_number ),
                        spell_color, splt.name.translated() );
             mvwprintz( w_level.window, point( 2, line_number++ ), spell_color,
-                       _( "%1$-3d/%2$3d" ), spell_level, splt.max_level );
+                       _( "%1$2d / %2$2d" ), spell_level, splt.max_level );
```

Comment:
right aligned numbers are easier to parse visually, but this leaves 3-digit spell levels (or negative with two digits) looking odd.